src/egon/data/db.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+from sqlalchemy import create_engine, text
+import yaml
+
+import egon
+
+
+def credentials():
+    """Return local database connection parameters.
+
+    Returns
+    -------
+    dict
+        Complete DB connection information
+    """
+    # Read database configuration from docker-compose.yml
+    package_path = egon.data.__path__[0]
+    docker_compose_file = os.path.join(
+        package_path, "airflow", "docker-compose.yml"
+    )
+    docker_compose = yaml.load(
+        open(docker_compose_file), Loader=yaml.SafeLoader
+    )
+
+    # Select basic connection details
+    docker_db_config = docker_compose["services"]["egon-data-local-database"][
+        "environment"
+    ]
+
+    # Add HOST and PORT
+    docker_db_config_additional = docker_compose["services"][
+        "egon-data-local-database"
+    ]["ports"][0].split(":")
+    docker_db_config["HOST"] = docker_db_config_additional[0]
+    docker_db_config["PORT"] = docker_db_config_additional[1]
+
+    return docker_db_config
+
+
+def execute_sql(sql_string):
+    """Execute a SQL expression given as string.
+
+    The SQL expression passed as plain string is convert to a
+    `sqlalchemy.sql.expression.TextClause`.
+
+    Parameters
+    ----------
+    sql_string : str
+        SQL expression
+
+    """
+    db_config = credentials()
+
+    engine_local = create_engine(
+        f"postgresql+psycopg2://{db_config['POSTGRES_USER']}:"
+        f"{db_config['POSTGRES_PASSWORD']}@{db_config['HOST']}:"
+        f"{db_config['PORT']}/{db_config['POSTGRES_DB']}",
+        echo=False,
+    )
+
+    with engine_local.connect().execution_options(autocommit=True) as con:
+        con.execute(text(sql_string))
+
+
+def submit_comment(json, schema, table):
+    """Add comment to table.
```

Comment:
Dunno :thinking:

Reply:
I was asking because it looks like that's the case, but I'm not too sure, too. If you don't know either, it might be a good candidate for a test, i.e. try out the behaviour first and then pretend it's intentional by writing a test which checks it. Fortunately, Travis [supports docker](https://docs.travis-ci.com/user/docker/).

Reply:
I moved it to an issue: https://github.com/openego/eGon-data/issues/51